### ISSUE TITLE
Training compress and decompress chair.txt

### ISSUE THREAD

**Issue by 9B8DY6** [daniel03c1/masked_wavelet_nerf#7]
I train chair txt and compress it. The name of ckpt is tensorf_lego_VM.th. After decompression, I get poor PSNR as 13.297615 in evaluation.
`python3 train.py --config=configs/chair.txt --use_mask --mask_weight=1e-10 --grid_bit=8 --use_dwt --dwt_level=4`
`python3 compress.py --compress=1 --compress_levelwise=1 --ckpt=~/tensorf_lego_VM/tensorf_lego_VM.th`
`python3 compress.py --decompress=1 --decompress_levelwise=1 --config=configs/chair.txt --ckpt=~/tensorf_lego_VM/tensorf_lego_VM.th`
The commands above is what I used.

**Comment by daniel03c1** [comment]
Thank you for your interest in our work. We accidentally omitted the Inverse DWT during the commit, but it has now been fixed, thanks to #6. If you encounter the same problem even after pulling the newest commit, please let me know.

**Comment by 9B8DY6** [comment]
@daniel03c1 you mean this one?
![image](https://github.com/daniel03c1/masked_wavelet_nerf/assets/67573223/5b6d27b6-3e51-40f7-b73a-34914d04b6e9)

**Comment by daniel03c1** [comment]
No. Not that one. Would you please check if the codes are up to date?

**Comment by 9B8DY6** [comment]
@daniel03c1 I am so sorry that I can not recognize what is changed in your github code.

**Comment by daniel03c1** [comment]
Could you please execute a 'git pull' and let me know if you still face the same problem again?

**Comment by daniel03c1** [comment]
@9B8DY6 If you are still having the same problem, even after you pull the latest commit, please let me know.
